src/util/tracking.ts

```diff
@@ -46,6 +47,7 @@ export interface TrackingValues {
   installerId?: string // Account installerId, i.e. referralId
   orderId?: string // Unique order identifier provided by plugin
   pluginId?: string // Plugin that provided the conversion
+  numSelectedWallets?: number // Number of wallets to be created
```

Comment:
`numCreatedWallets` might be a better name sense `TrackingValues` looks like a top-level namespace for our telemetry system.